src/components/header/Header.tsx

```diff
@@ -26,35 +24,40 @@ const Header = () => {
 
     //change id of the film for film page
     if (filmIndex > -1) {
-      if (films) dispatch(changeId(films[filmIndex].id));
+      //clean input field
+      setInputValue("");
+      if (films)
+        //jump to the film page
+        history.push(`/film-page/${films[filmIndex].id}`);

```

Comment:
where is that history variable coming from? is it global? don't react have methods for navigating to certain path?